sklearn/metrics/scorer.py

```diff
@@ -313,12 +322,12 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 
 # Standard regression scores
 r2_scorer = make_scorer(r2_score)
-mean_squared_error_scorer = make_scorer(mean_squared_error,
-                                        greater_is_better=False)
-mean_absolute_error_scorer = make_scorer(mean_absolute_error,
-                                         greater_is_better=False)
-median_absolute_error_scorer = make_scorer(median_absolute_error,
-                                           greater_is_better=False)
```

Comment:
I wouldn't want to make the wrapper to complex, but I feel that deprecating the call method is really the easiest thing. We also don't need to change the wrapper for that. It will then say ``__call__`` is deprecated and we can let the message explain that.